handlers/dimensions.go

```diff
@@ -1,27 +1,114 @@
 package handlers
 
 import (
+	"fmt"
 	"net/http"
+	"net/url"
+	"strings"
 
+	"github.com/ONSdigital/dp-api-clients-go/v2/filter"
 	"github.com/ONSdigital/dp-frontend-filter-flex-dataset/mapper"
 	"github.com/ONSdigital/dp-net/v2/handlers"
+	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/gorilla/mux"
 )
 
 // DimensionsSelector Handler
-func DimensionsSelector(rc RenderClient) http.HandlerFunc {
+func DimensionsSelector(rc RenderClient, fc FilterClient, dc DimensionClient) http.HandlerFunc {
 	return handlers.ControllerHandler(func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-		dimensionsSelector(w, req, rc, lang)
+		dimensionsSelector(w, req, rc, fc, dc, collectionID, accessToken, lang)
 	})
 }
 
-func dimensionsSelector(w http.ResponseWriter, req *http.Request, rc RenderClient, lang string) {
+func dimensionsSelector(w http.ResponseWriter, req *http.Request, rc RenderClient, fc FilterClient, dc DimensionClient, collectionID, accessToken, lang string) {
+	ctx := req.Context()
 	vars := mux.Vars(req)
+	filterID := vars["filterID"]
+	nameParam := vars["name"]
+
+	logData := log.Data{
+		"filter_id":  filterID,
+		"name_param": nameParam,
+	}
```

Comment:
Is it worth logging URI data?

Reply:
Removed the name param, kept the filter ID for consistency with the other handler.